delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/JsonLdValue.scala

```diff
@@ -24,4 +29,18 @@ object JsonLdValue {
       override val value: A                  = v
       override val encoder: JsonLdEncoder[A] = implicitly[JsonLdEncoder[A]]
     }
+
+  implicit val jsonLdEncoder: JsonLdEncoder[JsonLdValue] = {
+    new JsonLdEncoder[JsonLdValue] {
+      override def context(value: JsonLdValue): ContextValue                                                       = value.encoder.context(value.value)
+      override def expand(
+          value: JsonLdValue
+      )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, ExpandedJsonLd]  =
+        value.encoder.expand(value.value)
+      override def compact(
+          value: JsonLdValue
+      )(implicit opts: JsonLdOptions, api: JsonLdApi, rcr: RemoteContextResolution): IO[RdfError, CompactedJsonLd] =
+        value.encoder.compact(value.value)
+    }
+  }
```

Comment:
`JsonLdValue` is a slightly strange class. I can see why we have it, but I think also this typeclass makes sense for it. The alternative is that we do some wrangling in ResponseToJsonLd, but I think this makes more sense